crates/tabby/src/serve.rs

```diff
@@ -122,7 +122,10 @@ pub async fn main(config: &Config, args: &ServeArgs) {
 
     #[cfg(feature = "ee")]
     let (api, ui) = if args.webserver {
-        tabby_webserver::public::attach_webserver(api, ui, logger, code, config).await
+        let address = format!("http://{}:{}", args.host, args.port);
+        let (api, ui) =
+            tabby_webserver::public::attach_webserver(api, ui, logger, code, config, address).await;
```

Comment:
please only pass in the port (name it as `local_listen_port`), thus we could ensure we'll only spawn job locally for now.